templates/common/infra/bicep/core/search/search-services.bicep

```diff
@@ -1,4 +1,4 @@
-metadata description = 'Creates an Azure Cognitive Search instance.'
+metadata description = 'Creates an Azure AI Search instance.'
```

Comment:
do we need to update the azd progress reporter?   I think we are using `"Search service"` for `Microsoft.Search/searchServices`...  Is that the same?

Reply:
Search service seems fine, its agnostic, but you could be specific and call it AI Search service.